pagopa/io-app
# chore: [IOBP-141] Fix CGN_MERCHANT_DETAILS scrollView padding

## Short description
fixed ScrollView in `CGN_MERCHANTS_DETAIL` route

| BEFORE | AFTER |
|--------|--------|
|<img width="200" src="https://github.com/pagopa/io-app/assets/60693085/553c9e02-9861-4d77-80ab-a69f57ffe298">|<img width="200" alt="Screenshot 2023-07-24 at 11 58 11" src="https://github.com/pagopa/io-app/assets/60693085/fdbbf797-cad0-4cca-a602-607efe3a3834">|

## List of changes proposed in this pull request
- moved SafeAreaView to avoid wrong spacing 
- removed useless `React.Fragment`


## How to test
navigate to said screen and make sure both the success and error screen render correctly without hiding any content or preventing user interaction
